config/routes.rb

```diff
@@ -24,7 +24,7 @@
   devise_for :users, controllers: { omniauth_callbacks: "users/omniauth_callbacks" }
 
   devise_scope :user do
-    get "sign_in", to: "devise/sessions#new", as: :new_user_session
+    get "sign_in", to: "home#index"
     get "sign_in", to: "users/omniauth_callbacks#passthru", as: :session
```

Comment:
Why do we have two `get "sign_in"` routes? 

How does Rails decide which one to use (i.e. `home#index` vs `users/omniauth_callbacks#passthru`). 

@kelynch we already had two before your PR but you changed one to go to `home#index` so I figure you must have a reason.